cmd/kubeadm/app/cmd/phases/reset/cleanupnode.go

```diff
@@ -84,10 +85,15 @@ func runCleanupNode(c workflow.RunData) error {
 		// Try to unmount mounted directories under kubeadmconstants.KubeletRunDirectory in order to be able to remove the kubeadmconstants.KubeletRunDirectory directory later
 		fmt.Printf("[reset] Unmounting mounted directories in %q\n", kubeadmconstants.KubeletRunDirectory)
 		// In case KubeletRunDirectory holds a symbolic link, evaluate it
-		kubeletRunDir, err := absoluteKubeletRunDirectory()
+		kubeletRunDir, retainDirs, err := absoluteKubeletRunDirectory()
 		if err == nil {
 			// Only clean absoluteKubeletRunDirectory if umountDirsCmd passed without error
-			dirsToClean = append(dirsToClean, kubeletRunDir)
+			if r.DryRun() {
```

Comment:
Dead code here?
The `if r.DryRun()` is in the parent `if !r.DryRun()` block.

Reply:
sorry, typo